torch/_prims/context.py

```diff
@@ -170,13 +151,23 @@ def __torch_function__(
         mapping = torch_to_refs_map()
         func = mapping.get(orig_func, None)
 
+        skip_decomposition_table: Sequence[
```

Comment:
Probably wouldn't want to define this datastructure in the function? Also would need a comment describing it